docs/pages/reference/metrics.mdx

```diff
@@ -2,12 +2,38 @@
 title: Teleport Metrics
 description: Comprehensive list of all metrics exposed by Teleport.
 ---
-The following metrics are available:
 
 <Notice scope={["cloud"]} type="tip">
 
 Teleport Cloud does not expose monitoring endpoints for the Auth Service and Proxy Service.
 
 </Notice>
 
+Teleport metrics data is primarily used for performance monitoring. If you'd like to
```

Comment:
Does "primarily" mean that there is another use? Should we say "Teleport metrics are intended for performance monitoring"?

Reply:
> "Teleport metrics are intended for performance monitoring" 

Correct... It's intended to be used for performance monitoring, but you could be using it for any number of things. Let's go with this. I'll adjust the PR.